src/Elliptic/Systems/Poisson/Equations.hpp

```diff
@@ -59,6 +59,17 @@ void euclidean_fluxes(
     gsl::not_null<tnsr::I<DataVector, Dim, Frame::Inertial>*> flux_for_field,
     const tnsr::i<DataVector, Dim, Frame::Inertial>& field_gradient) noexcept;
 
+/*!
+ * \brief Compute the fluxes \f$F^i=\sqrt{\gamma}\gamma^{ij}\partial_j u(x)\f$
+ * for the curved-space Poisson equation on a metric \f$\gamma_{ij}\f$.
+ */
+template <size_t Dim>
+void noneuclidean_fluxes(
+    gsl::not_null<tnsr::I<DataVector, Dim, Frame::Inertial>*> flux_for_field,
+    const tnsr::II<DataVector, Dim, Frame::Inertial>& inv_metric,
```

Comment:
Is this the metric or spatial metric? Your code says metric but your dox say spatial metric.

Reply:
pushed a fixup, please take a look